core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -37,16 +49,23 @@
           </editor-html-field>
         </div>
       </div>
+
+      <div ng-show="chooseQuestionType" class="question-type-wrapper" ng-if="canAddNewQuestion()">
```

Comment:
The original mock defaulted to selecting the multiple-choice input question type, and allowed the user to change it via a dropdown. I seem to recall us discussing this a while back, but just to recap, what is the rationale for not following the mock?